tephra-hbase-compat-0.96/src/main/java/co/cask/tephra/hbase96/TransactionAwareHTable.java

```diff
@@ -579,6 +581,9 @@ private Delete transactionalizeAction(Delete delete) throws IOException {
         }
       }
     }
+    for (Map.Entry<String, byte[]> entry : delete.getAttributesMap().entrySet()) {
+        txDelete.setAttribute(entry.getKey(), entry.getValue());
+    }
```

Comment:
In addition to adding attributes, `transactionalizeAction(Put put)` also copies over `setWriteToWAL` flag. To keep the behavior consistent, can you please add that over here too?